src/profile/notification-settings/NotificationSettings.ts

```diff
@@ -0,0 +1,184 @@
+    ]),
+    div({ style: { display: 'flex', justifyContent: 'center' } }, [
+      boldDiv('Changed Opt In'),
+      h(InfoBox, { style: { marginLeft: '0.5rem' } }, [workspaceChangedNotificationInfo]),
+    ]),
+  ]);
+});
+
+export const NotificationSettings = () => {
+  const { workspaces } = useWorkspaces();
+  const [prefsData] = _.over(_.pickBy)((_v, k) => _.startsWith('notifications/', k), userStore.get().profile);
+  const [saving, setSaving] = useState(false);
+
+  const userAttributes = userStore.get().terraUserAttributes;
+  const [marketingConsent, setMarketingConsent] = useState(userAttributes.marketingConsent);
+
+  useEffect(() => {
+    setMarketingConsent(userAttributes.marketingConsent);
+  }, [userAttributes.marketingConsent]);
+
+  return h(PageBox, { role: 'main', style: { flexGrow: 1 }, variant: PageBoxVariants.light }, [
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, margin: 0, textTransform: 'uppercase' } }, [
+        'System Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
+          'You may opt in or out of recieving marketing communications from Terra',
+        ]),
+      ]),
+    ]),
+    h(NotificationCardHeaders),
+    div({ role: 'list', 'aria-label': 'communication preferences from terra', style: { flexGrow: 1, width: '100%' } }, [
+      h(UserAttributesCard, {
+        value: true,
+        label: 'Necessary communications related to platform operations',
+        setSaving,
+        notificationKeys: [],
+        notificationType: 'PlatformOperations',
+        disabled: true,
+      } as UserAttributesCardProps),
+      h(UserAttributesCard, {
+        value: marketingConsent,
+        label: 'Marketing communications (including upcoming workshops and new flagship dataset additions)',
+        setSaving,
+        notificationKeys: ['notifications/MarketingConsent'],
+        notificationType: 'Marketing',
+      } as UserAttributesCardProps),
+    ]),
+    div({ style: Style.cardList.toolbarContainer }, [
+      h2({ style: { ...Style.elements.sectionHeader, marginTop: '2rem', textTransform: 'uppercase' } }, [
+        'Account Notifications',
+        h(InfoBox, { style: { marginLeft: '0.5rem' } }, [
+          'You may receive email notifications regarding certain events in Terra. You may globally opt in or out of receiving these emails.',
+        ]),
+      ]),
+    ]),
+    h(NotificationCardHeaders),
+    div(
+      { role: 'list', 'aria-label': 'notification settings for your account', style: { flexGrow: 1, width: '100%' } },
+      [
+        h(NotificationCard, {
+          setSaving,
+          prefsData,
+          label: 'Group Access Requested',
+          options: [
+            {
+              notificationKeys: ['notifications/GroupAccessRequestNotification'],
+              notificationType: 'GroupAccessRequest',
+              optionLabel: 'Opt-in to group access requested notifications',
```

Comment:
The aria labels for checkboxes are now more explicit. This was especially needed for workspace-level notifications, since we have 2 types now.